### PR TITLE
Fixes OpenSearch version reterival logic

### LINKED ISSUE #2188: Performance impacted during data ingestion
**Describe the bug**
In the OpenSearch datastore helper module in particular import_event function. a call is made to the function responsible for fetching the running OpenSearch version every time an event is added. this dramatically increases the data ingestion time.

https://github.com/google/timesketch/blob/db986692f1f73614290ba9116f0d28d6b1c01062/timesketch/lib/datastores/opensearch.py#L981

**To Reproduce**
1. Import data into Timesketch

**Expected behavior**
The version information should be retrieved once when the OpenSearch connection is first initialized.

### PR DESCRIPTION
**Pull request by rushattac**
This pull request fixes  #2188 by changing the way the OpenSearch version information is retrieved. 

+ What existing problem does this PR solve? This PR solves an issue where the version information for OpenSearch is retrieved for every event imported.
+ What new feature is being introduced with this PR? no new features.
+ Overview of changes to existing functions if required. None

**Checks**

- [ ] All tests succeed.
- [ ] Unit tests added.
- [ ] e2e tests added.
- [ ] Documentation updated.

**Closing issues**

closes #2188

### CODE REVIEW

File: timesketch/lib/datastores/opensearch.py [hunk 1]
```diff
@@ -1104,14 +1105,4 @@ def flush_queued_events(self, retry_count=0):
...
-    @property
-    def version(self):
-        """Get OpenSearch version.
-
-        Returns:
-          Version number as a string.
-        """
-        version_info = self.client.info().get("version")
-        return version_info.get("number")
+        return return_dict
```
Comment (berggren) [inline timesketch/lib/datastores/opensearch.py:1108]:
```suggestion
        return return_dict

```

**Comment by berggren** [comment]
Please take a look at the failing unit tests and see what needs to be done.

**Comment by rushattac** [comment]
> Please take a look at the failing unit tests and see what needs to be done.

hello @berggren should i create a mock for OpenSearch info request (might take some times as i am not familiar with unittests )
or  just wrap the code with try and except.

**Review by berggren — DISMISSED** [review]

**Comment by rushattac** [comment]
hi @berggren the unittest are now passing. please review, also now that the project is using OpenSearch and the included docker container is pinned to version 1.3.2 does it make sense to maintain backwards compatibility with ElasticSearch version 6?

**Comment by berggren** [comment]
> 

Thanks, I'll take a look and merge. Regarding support for ES 6.x: This is something we will take a look at later this year and drop support as necessary.

**Review by berggren — APPROVED** [review]

**Pull request merged**